src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/component/pact/provider/ManageRecallsUIPactTest.kt

```diff
@@ -236,6 +244,16 @@ class ManagerRecallsUiAuthorizedPactTest : ManagerRecallsUiPactTestBase() {
     `a note exists`(2)
   }
 
+  @State(
+    "a recall with part B records exists",
+    "a Fixed Term recall with part B records exists",
```

Comment:
not sure this makes sense. Isn't Part B only for Standard recalls?

Reply:
Will bin.  TBH I'm not convinced we need any of these "a Fixed Term recall blah ..." states?

Reply:
I think only reason to have them is if there is something specific about the state which means it must be Fixed/Standard. but agree, possibly gone overkill